docs/discover/context.asciidoc

```diff
@@ -0,0 +1,63 @@
+[[discover-document-context]]
+== View surrounding documents
+
+Once you've narrowed your search to a specific event in *Discover*,
+you can inspect the documents that occurred
+immediately before and after the event.
+To view the surrounding documents, your index pattern must contain time-based events.
+
+. In the document table, click the expand icon (>).
+. Click *View surrounding documents.*
++
+In the context view, documents are sorted by the time field specified in the index pattern
+and displayed using the same set of columns as the *Discover* view from which
+the context was opened. The anchor document is highlighted in blue.
++
+[role="screenshot"]
```

Comment:
I think we should change the context view to show the anchor doc in the middle, so the reader sees docs both above and below the anchor.

Reply:
👍 I've changed it new file is also smaller in size but better in resolution, so it's a win
<img width="1241" alt="discover-context" src="https://user-images.githubusercontent.com/463851/109787838-eea35500-7c0e-11eb-87d5-32d8aa62808a.png">